src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -179,6 +181,31 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_commit_sha(
+        self,
+        integration: Integration,
+        organization_id: str,
+        line_no: int,
+        filepath: str,
+        current_config: JSONData,
+    ) -> Optional[str]:
+        commitSha = None
+        githubIntegration = GitHubIntegration(integration, organization_id)
```

Comment:
I believe you can do this like this:
`client = integration.get_installation(organization_id=organization.id).get_client()`

That way it becomes provider independent.